examples/README.md

```diff
@@ -233,15 +233,15 @@ The cmake method has an option to build and run tests as parallel jobs. This can
 You can specify the number of jobs to build and run tests in parallel:
 
 ```bash
-./run.py -c --j 4
+./run.py -c -j 4
 ```
 
-Another option is to use a different build system with system that is designed for speed like [Ninje](https://ninja-build.org) which automatically maximizes its use of available cores on your system.
+Another option is to use a different build system with system that is designed for speed like [Ninja](https://ninja-build.org) which automatically maximizes its use of available cores on your system.
 
 If you have `Ninja` installed, you can have cmake use it your underlying build system and automatically get improved performance vs the default Make-based system:
 
 ```bash
-./runtests.py -c -gNinja
+./runt.py -c -gNinja
```

Comment:
Hi @shakfu !

Shouldn't that be `run.py` as well? I would do the change if I was sure, but I'm not :)

Thanks!

Reply:
Yes, you are right. I had the same thought precisely. The change is already committed in my last commit (-: